snafu/benchmarks/coremarkpro/coremarkpro.py

```diff
@@ -0,0 +1,262 @@
+# /usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""Runs CoreMark Pro."""
+import re
+import shlex
+import uuid
+from datetime import datetime
+from typing import Dict, Iterable, List
+
+from dateutil import tz
+
+from snafu.benchmarks import Benchmark, BenchmarkResult
+from snafu.config import ConfigArgument
+from snafu.process import sample_process
+
+
+class Coremarkpro(Benchmark):
+    """Wrapper for CoreMark Pro"""
+
+    # Set for Benchmark Class
+    tool_name = "coremark-pro"
+    args = (
+        ConfigArgument(
+            "-p",
+            "--path",
+            dest="path",
+            type=str,
+            help="Path to coremark-pro's directory",
+            required=True,
+        ),
+        ConfigArgument(
+            "-c",
+            "--context",
+            dest="context",
+            type=int,
+            help="CoreMark Pro's context",
+            default=1,
+            required=False,
+        ),
+        ConfigArgument(
+            "-w",
+            "--workloads",
```

Comment:
docs say --workload and this is --workloads. Not sure which one is correct.

Reply:
Good catch, fixed, after looking back at the docs, it was neither and was `worker`.